services/brig/test/integration/API/SystemSettings.hs

```diff
@@ -0,0 +1,42 @@
+module API.SystemSettings (tests) where
+
+import Bilge
+import Bilge.Assert
+import Brig.Options
+import Control.Lens
+import Control.Monad.Catch
+import Imports
+import Test.Tasty
+import Test.Tasty.HUnit
+import Util
+import Wire.API.SystemSettings
+
+tests :: Opts -> Manager -> Brig -> IO TestTree
+tests opts m brig = pure $ do
+  testGroup
+    "settings"
+    [ test m "GET /system/settings" $ testGetSettings opts brig
+    ]
+
+testGetSettings :: Opts -> Brig -> Http ()
+testGetSettings opts brig = liftIO $ do
+  expectResultForSetting Nothing False
+  expectResultForSetting (Just False) False
+  expectResultForSetting (Just True) True
+  where
+    expectResultForSetting :: Maybe Bool -> Bool -> IO ()
+    expectResultForSetting s expectedRes = do
```

Comment:
Please avoid single-byte identifiers in the future!  (I know, it's a very short function, and I'm not doing this consistently either, but still...)

Reply:
Agreed :+1: